.github/workflows/serverless-integration.yml

```diff
@@ -32,9 +35,19 @@ jobs:
           repository: DataDog/datadog-lambda-extension
           path: go/src/github.com/DataDog/datadog-lambda-extension
 
+      - name: Set up QEMU
```

Comment:
Why do we need QEMU here?

Reply:
Why do we need to run arm64 images on amd64 machines? My understanding is that the tests run on actual AWS infrastructure -- it shouldn't matter what the architecture we use to run the test script. What made you decide to add QEMU here? What error do you see if you remove it?

Reply:
Not saying we don't need it, just curious why

Reply:
Sorry, I was wrong about the "run" part. QEMU is a "process emulator" that simulates an arm64 environment so that docker buildx can build the arm64 image. Without QEMU, docker fails to build the image, heres a run I did with QEMU removed: https://github.com/DataDog/datadog-agent/runs/5411036221?check_suite_focus=true.

We have the error: `ERROR: process "/dev/.buildkit_qemu_emulator /bin/sh -c apt-get install -y zip" did not complete successfully`, it seems like QEMU is a requirement for docker buildx to build an arm64 image on a amd64 machine, so it's expecting it to be installed.

Reply:
Thanks for the detailed explanation! 🙏